frontend/src/layout/navigation-3000/sidepanel/SidePanel.tsx

```diff
@@ -71,12 +81,17 @@ export const SIDE_PANEL_TABS: Record<SidePanelTab, { label: string; Icon: any; C
         Icon: IconConfetti,
         Content: SidePanelWelcome,
     },
+    [SidePanelTab.Status]: {
+        label: 'Status',
+        Icon: IconLive,
```

Comment:
A bit of overloading going on here, since we already use this icon for Events (aka Live events). My suggestion: a "cloud" icon. Below comes from Central icons, so we should put it in @posthog/icons, but you can demo with the SVG locally

![image](https://github.com/PostHog/posthog/assets/4550621/dd4a44f8-7185-4d15-8d33-939058d74423)

```html
<svg width="24" height="24" viewBox="0 0 24 24" fill="none" xmlns="http://www.w3.org/2000/svg">
<path d="M17.125 19.25H8.925C4.96236 19.25 1.75 16.0041 1.75 12C1.75 7.99594 4.96236 4.75 8.925 4.75C11.5605 4.75 13.8642 6.18583 15.1116 8.32557C15.3741 8.77588 15.9058 9.01272 16.4221 8.94116C16.6519 8.90931 16.8865 8.89286 17.125 8.89286C19.9555 8.89286 22.25 11.2114 22.25 14.0714C22.25 16.9315 19.9555 19.25 17.125 19.25Z" stroke="black" stroke-width="1.5" stroke-linejoin="round"/>
</svg>
```